modules/ROOT/pages/upgrade-openshift.adoc

```diff
@@ -0,0 +1,10 @@
+= Upgrading Runtime Fabric on Red Hat OpenShift 
+
+To upgrade Runtime Fabric on Red Hat OpenShift, use the Runtime Fabric operator in the Red Hat OpenShift console.
+
+NOTE: Red Hat OpenShift console allows you to run upgrades automatically (default) or manually. You can change this setting in the *Subscription* tab in the Runtime Fabric Operator. 
```

Comment:
I'd leave this block just as text, and switch it with the first sentence, since this one introduces info about the upgrade process and the first one introduces the list of steps to do the task.

Reply:
ok